recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "debugpy" %}
-{% set version = "1.5.1" %}
+{% set version = "1.6.7" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
-  sha256: d2b09e91fbd1efa4f4fda121d49af89501beda50c18ed7499712c71a4bf3452e
+  sha256: c4c2f0810fa25323abfdfa36cbbbb24e5c3b1a42cb762782de64439c575d67f2
 
 build:
   number: 0
```

Comment:
`- python                                 # [build_platform != target_platform]` can be removed from `requirements/build` as we do not use cross-compilation

Reply:
It's been removed